README.md

```diff
@@ -29,15 +29,24 @@ This site relies heavily on node and npm.
 1. Make sure you are using at least node v.5+ and latest npm; 
 if not install [nvm](https://github.com/creationix/nvm) to get node going on your machine.
 
-1. Install npm `gulp` package *globally*: `npm install -g gulp`
+1. Install global npm packages by running `./scripts/before-install.sh`
```

Comment:
Sure, but wouldn't that apply to all scripts mentioned in the dev setup?

I'm just trying to keep this all DRY. We know that the scripts are kept up-to-date because they are run on Travis. If we duplicate the script commands here, then they run the risk of becoming out of step ... as they did this morning.

If a Windows user can't run a script I'd assume that they'd look inside and run the equivalent commands under Windows. Would that do?